sockeye/inference.py

```diff
@@ -900,8 +900,8 @@ def _decode_step(self,
             model_probs.append(probs)
             model_attention_probs.append(attention_probs)
             model_states.append(state)
-        probs, attention_probs = self._combine_predictions(model_probs, model_attention_probs)
-        return probs, attention_probs, model_states
+        neg_logprobs, attention_probs = self._combine_predictions(model_probs, model_attention_probs)
```

Comment:
would it make sense to rename the probs variable a couple of lines above as well?

Reply:
the conversion happens only in combine_predictions so these are actually probabilities and not logprobs. That's actually the reason I wanted to rename the probs after combine_predictions to make that clear.

Reply:
uh yes you are right!